draft-ietf-privacypass-auth-scheme.md

```diff
@@ -173,10 +173,10 @@ in an IANA registry, {{token-types}}. Challenges with unsupported token_type
 values MUST be ignored. This value determines the structure and semantics of
 the rest of the structure.
 
-- "issuer_name" is a string containing the name of the issuer. This is a
-hostname that is used to identify the issuer that is allowed to issue
-tokens that can be redeemed by this origin. The string is prefixed with a
-2-octet integer indicating the length, in network byte order.
+- "issuer_name" is a UTF-8 string containing the hostname of the issuer.
```

Comment:
Is it UTF-8 hostnames? eg Ulabel? not A-label (eg punycode) ?

If the unit these things operate on are "dns hostnames", then I would expect this to use A-labels. That is not nohâts.ca but xn--nohts-5qa.ca

Reply:
It's just a FQDN without the trailing dot, e.g., "github.com" or "ietf.org". Basically, the same thing that would go in the TLS SNI. It's _not_ an A-label with the bizarre xn-- prefix. I'll try to clarify.

Reply:
@paulwouters how does the new text look?

Reply:
Yeah, this is the HTTP / URI style version of the hostname, not the DNS version of the name.